<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>701</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Up">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>前</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>后</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Right">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ParamSet">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>720</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>699</width>
       <height>149</height>
      </rect>
     </property>
     <widget class="QLabel" name="LabelHis">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X轴复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Punch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>冲压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seperate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>离合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Press">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>冲压一次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Blow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>吹气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>730</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计数清零</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Qty">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>730</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Run">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ModelSet">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>720</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模板设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FuryAI视觉送料机</string>
    </property>
   </widget>
   <widget class="QLabel" name="Labell_Version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前版本：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Min">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Limit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>730</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Plus">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>730</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Miner">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>730</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
